theme/scripts/utils/isInViewport.ts

```diff
@@ -0,0 +1,11 @@
+export default (element: Element) => {
+  const rect = element.getBoundingClientRect();
+  const threshold = 2;
```

Comment:
Would be cool if threshold was a param, so we could easily tweak it from wherever it is getting called